### DISCUSSION TITLE
Safe Exam Browser 3.8.0 Crashing - 3.7 and 3.6 works

### DISCUSSION THREAD

**Discussion by Virvelvind** [SafeExamBrowser/seb-win-refactoring#1044] [Q&A]
2 users are not able to launch the newest version of SEB, and they need it for an exam in 14 days. They both have Elitebook 840 G9
We have ruled out antivirus, webview, runtime etc. 

This is the error message they get when trying to open SEB, they also get error when trying to open SEB config

An internal exception occurred ( Address: 0x40463459

The eventlog in Windows shows absolutely nothing related to the app crash, it's just like it never happend. Minidump also shows nothing. The SEB logfile also shows nothing.

![image](https://github.com/user-attachments/assets/1ca2218f-3950-4ae8-b55a-8db7faf601f0)

**Comment by dbuechel** [comment]
Please specify your exact use case, the SEB and OS version information and provide all of the above mentioned log files. From what I recall, we do not have any such error message in the entirety of our code base...

**Reply by Virvelvind** [reply]
2 users are going to take an exam soon. They are unable to open the new SEB version at all. They instantly get the error message i posted. When i checked the SEB logfiles they showed no info inside them at all. I can try to double check tomorrow and upload if theres anything. They also instantly  get an error when trying to open the seb configure tool aswell.  Both are up to date on drivers, bios, os, firmware. Tried installing visual basic/.net manually. no help.

**Reply by dbuechel** [reply]
> When i checked the SEB logfiles they showed no info inside them at all. I can try to double check tomorrow and upload if theres anything.

Yes please, that would be great.

> They also instantly get an error when trying to open the seb configure tool aswell.

That is even more suspicious. Please try to run the [SEB Verificator](https://github.com/SafeExamBrowser/seb-win-verificator) on those systems and check whether these students are really using an official release version and not possibly a custom or manipulated build of Safe Exam Browser.

**Reply by dbuechel** [reply]
Also, please specify the OS version information (e.g. Windows 10 Pro 22H2).

**Reply by Virvelvind** [reply]
They downloaded it from the exam "demo" link. I also tried via remote support to download straigth from the web website. Works great for me on my elitebook, but not on theirs, and they habe exactly the same setup as us. In worst case scenario, is it impossible for them to use the 3.7 version that works for them? When they try the demo exam they get prompted to download 3.8 to be able to use the exam

**Reply by dbuechel** [reply]
> In worst case scenario, is it impossible for them to use the 3.7 version that works for them? When they try the demo exam they get prompted to download 3.8 to be able to use the exam.

If there is a version restriction, then no, that's not possible. But they may be able to use the 32-bit build (..._x86_Setup.msi); if so, please do let me know that using the 32-bit build resolves the issue for them.

**Comment by Virvelvind** [comment]
When i ran the SEB application there was no generated logfile in appdata/local/safeexambrowser location. I then ran the Verificator app with x64 installed, but also with the x86 version installed, they actually gave different errors when trying to open them, but both failed. The logfile has both x64 and x86 errors. Hopefully we can try to figure out whats happening.

[2024-12-03.log](https://github.com/user-attachments/files/17993618/2024-12-03.log)

**Reply by dbuechel** [reply]
According to the log, the 64-bit as well as the 32-bit builds were successfully verified (thus not manipulated). It only appears that starting the 32-bit build was not possible due to an error. Is the student using any particular security / anti-virus software or even Crowdstrike?

**Comment by Virvelvind** [comment]
They are using Cylance antivirus delivered and administered by us, but its only in report mode while troubleshooting. I have almost same hardware and we are also using cylance ourself, and it works fine on my Elitebook 850 G8 on win 11.

**Reply by dbuechel** [reply]
Then you may have any luck going through all running processes on that system and terminating any potentially interfering applications (e.g. other lockdown browsers, remote connection software or alike). But again, I myself have never seen this error message and am confident that it is not part of our code base itself - if I could reproduce the issue, I might be able to identify its origin...
